### DISCUSSION TITLE
react-instantsearch-hooks has no refresh feature

### DISCUSSION THREAD

**Discussion by kanehara** [algolia/instantsearch#5359] [Ideas]
react-instantsearch-core provides a `refresh` prop in the `InstantSearch` component to allow for manual refresh of Algolia results - e.g. pull down to refresh.  `react-instantsearch-hooks` however is missing this feature which would be nice to have.  I also have not found any workaround to mimic the old `refresh` prop.

**Comment by francoischalifour** [comment]
I explain the `refresh` situation and solution in this thread: https://github.com/algolia/react-instantsearch/discussions/3483#discussioncomment-2832728.
